test/es-module/test-esm-command.mjs

```diff
@@ -0,0 +1,31 @@
+import '../common/index.mjs';
+import { spawnSync } from 'node:child_process';
+import { strictEqual } from 'node:assert';
+import { fileURLToPath } from 'node:url';
+
+{
+  const child = spawnSync(process.execPath, [
+    '--input-type',
+    'module',
+    '-e',
+    'console.log(import.meta.command)',
+  ]);
+
+  if (child.status !== 0) {
+    console.error(child.stderr.toString());
+  }
+
+  strictEqual(child.stdout.toString().trim(), 'true');
+}
+
+{
+  const child = spawnSync(process.execPath, [
+    fileURLToPath(new URL('../fixtures/es-modules/command-main.mjs', import.meta.url)),
+  ]);
+
+  if (child.status !== 0) {
+    console.error(child.stderr?.toString() ?? child.error);
+  }
+
+  strictEqual(child.stdout.toString().trim(), 'ok');
+}
```

Comment:
What about `node --import a.js b.js`? Which of `a.js` or `b.js` get `import.meta.command`?

Likewise, what about `node --require a.cjs --import b.js c.js`?

Reply:
`b.js` in the first example and `c.js` in the second example are the "CLI Command" that is being executed.

Reply:
Besides documenting our intent, I feel like we should have some tests that include `--require` and `--import` arguments; and workers.